nilearn/glm/contrasts.py

```diff
@@ -386,6 +428,13 @@ def compute_fixed_effects(contrast_imgs, variance_imgs, mask=None,
         Whether fixed effects estimates should be weighted by inverse
         variance or not. Default=False.
 
+    dofs : array-like, with len = len(variance_imgs) False or None, default=None
+        the degrees of freedom of the models
+        when None, it is assumed that the degrees of freedom are 100 per input.
```

Comment:
```suggestion
    dofs : array-like, False or None, default=None
        the degrees of freedom of the models with len = len(variance_imgs)
        when None, it is assumed that the degrees of freedom are 100 per input.
```

Reply:
An additional comment, dofs=False is not tested and I think will fail. What do we want out of this option?